docs/src/man/getting_started.md

```diff
@@ -785,6 +785,56 @@ not be mutated without caution.
 The exact rules of handling columns of a `DataFrame` are explained in
 [The design of handling of columns of a `DataFrame`](@ref man-columnhandling) section of the manual.
 
+## Replacing Data
+
+Use the broadcasting syntax to apply replacement operations to data frames.
+
+Note: in-place replacement in a data frame requires the type of the replacement value match the respective column data type.
```

Comment:
I'd first show a simple case which doesn't require `allowmissing!`, with an example code block, and then the more complex case.